Marlin/src/gcode/config/M217.cpp

```diff
@@ -22,8 +22,6 @@
 
 #include "../../inc/MarlinConfigPre.h"
 
-#if ENABLED(SINGLENOZZLE)
```

Comment:
Making `M217` always enabled even on machines that have only a single extruder makes no sense.